server/middleware/populateCurrentUser.ts

```diff
@@ -28,6 +29,14 @@ export default function populateCurrentUser(): RequestHandler {
         res.locals.user.staffId = parseInt(userId, 10) || undefined
       }
 
+      if (res.locals.user.authSource === 'delius') {
+        // Assemble user information from Delius
+        const probationUser = await userService.getStaffDetailsByUsername(req?.middleware?.clientToken, res.locals.user)
+
```

Comment:
Assigning staff Id and code for now and then we can add accordingly as required.
Don't want to add unused params.